app/routes/exercises.rb

```diff
@@ -59,6 +59,18 @@ class Exercises < Core
         redirect '/dashboard'
       end
 
+      post '/exercises/delete' do
+        exercises = UserExercise.find(params['exercise_ids'])
```

Comment:
We have a bit of a problem with names throughout the app. UserExercises are called exercises on the user (see `lib/exercism/user.rb:11`)

I agree that this should be scoped to the current user `current_user.exercises.find`